pkg/provider/github/acl.go

```diff
@@ -71,8 +71,16 @@ func (v *Provider) aclCheckAll(ctx context.Context, rev *info.Event) (bool, erro
 		return true, nil
 	}
 
-	// If the user who has submitted the pr is a owner on the repo then allows
-	// the CI to be run.
+	if rev.PullRequestNumber != 0 {
```

Comment:
Please can you readd the comment that was deleted and add another comment explaining what issue this return true fixes?
(make sure you specify this can only happen with github apps and bots)

Reply:
it wasn't before?

Reply:
nvm i got your point from this comment https://github.com/openshift-pipelines/pipelines-as-code/pull/1200#discussion_r1146179054

Reply:
Added comment